kubernetes/apply-monitoring.sh

```diff
@@ -1,79 +1,79 @@
 #!/bin/sh
 set -eu
 
-ROOT_DIR=$(cd $(dirname -- $0); pwd)
+ROOT_DIR="$(cd "$(dirname -- '$0')"; pwd)"
```

Comment:
Codacy found an issue: [Expressions don't expand in single quotes, use double quotes for that.](https://app.codacy.com/manual/yurake/k8s-3tier-webapp/pullRequest?prid=5892274)